src/data/businessCase.ts

```diff
@@ -96,19 +97,35 @@ export const prepareBusinessCaseForApp = (
     }
   });
 
+  // Fill in at least 1 entry if the year is empty so the UI can render
+  // TODO HACK idk it works
+  Object.values(lifecycleCostLines).forEach(solution => {
```

Comment:
If I'm correct, this may not be needed. If we're saving at least one line per year per solution, we'll get at least one line back when we fetch a business case. If we do need this, I think this is a perfectly nice way of implementing it.

Reply:
I see. Yeah we will need to revisit this. It looks like the backend is preventing the app from saving empty entries.